node/src/chain/acceptor.rs

```diff
@@ -1337,13 +1339,29 @@ pub(crate) async fn verify_block_header<DB: database::DB>(
     provisioners: &ContextProvisioners,
     header: &ledger::Header,
 ) -> Result<(u8, Vec<Voter>, Vec<Voter>), HeaderError> {
+    // Set the expected generator to the one extracted by Deterministic
+    // Sortition, or, in case of Emergency Block, to the Dusk Consensus Key
+    let (expected_generator, check_att) =
+        if is_emergency_block(header.iteration) {
+            let dusk_key = PublicKey::new(*DUSK_CONSENSUS_KEY);
+            let dusk_key_bytes = dusk_key.bytes();
+
+            // We disable the Attestation check since it's not needed to accept
+            // an Emergency Block
+            (*dusk_key_bytes, false)
```

Comment:
Do you need to add an exception when accepting the next block in order to not verify previous (missing) attestation?

Reply:
Yes. I've added the exception to `verify_prev_block_cert`